styx-common/src/test/java/com/spotify/styx/storage/DatastoreStorageTest.java

```diff
@@ -647,10 +651,17 @@ public void shouldStoreAndReadBackfill(String reverse) throws Exception {
         .nextTrigger(Instant.parse("2017-01-01T00:00:00Z"))
         .schedule(DAYS);
 
-    if (!reverse.isEmpty()) {
+    if (!reverse.trim().equals("_")) {
```

Comment:
Does `, true` actually fail so we need to do this trick or just to make it read nicer?

Reply:
felt fragile to rely on the parsing of `", true"`

Reply:
👍